fvm/src/kernel/default.rs

```diff
@@ -57,37 +44,26 @@ const TEST_ACTOR_ALLOWED_TO_CALL_CREATE_ACTOR: ActorID = 98;
 pub struct DefaultKernel<C> {
```

Comment:
I think we should consider renaming this to the "base kernel". But that can be in a subsequent PR to avoid bloating this one.

Reply:
See https://github.com/filecoin-project/ref-fvm/pull/1930